tests/FSharp.Stats.Tests/Testing.fs

```diff
@@ -348,6 +348,23 @@ let benjaminiHochbergTests =
     ]
 
 
+[<Tests>]
+let dunnSidakTest =
+    // taken from: https://www.real-statistics.com/hypothesis-testing/familywise-error/bonferroni-and-dunn-sidak-tests/
```

Comment:
The link is broken. Please provide references to papers, R, SPSS packages or similar sources